HW5/Anastasiia-web/1.py

```diff
@@ -0,0 +1,17 @@
+# Task1. In the range from 1 to 10 determine 

```

Comment:
x1 = []
x2 = []
x3 = []

for x in range(1, 11):
    if x % 2 == 0:
        #print(x, 'is even multiple of 2')
        x1.append(x)
    elif x % 3 == 0:
        #print(x, 'is an odd multiple of 3')
        x2.append(x)    
    else:
        #print(x, 'not divisible by 2 and 3')
        x3.append(x)
print(x1)        
print(x2)
print(x3)